rule_schema_v2.atd

```diff
@@ -352,6 +352,8 @@ type formula = {
 
   (* NEW: since 1.74 *)
```

Comment:
What about syntax v1?

Reply:
This file is just about syntax v2 (hence the filename).